app/components/cases/cases-filter/component.js

```diff
@@ -33,13 +36,19 @@ export default Component.extend({
 	}).restartable(),
 
 	actions: {
-		selectDateFrom(date) {
-			this.set('dateFrom', date);
-			this.searchTask.perform();
-		},
-		selectDateTo(date) {
-			this.set('dateTo', date);
-			this.searchTask.perform();
-		}
+    selectDateFrom(date) {
+      this.set('dateFrom', date);
+      this.searchTask.perform();
+    },
+    selectDateTo(date) {
+      this.set('dateTo', date);
+      this.searchTask.perform();
+    },
+    initTrigger() {
```

Comment:
Because these two actions actually do each other's opposite, it would be wise to also name them in such a way. So `openPopover` or `openLegend` would be better.